sdk/js/src/token_bridge/createWrapped.ts

```diff
@@ -114,3 +105,15 @@ export async function createWrappedOnNear(
   msgs.push({ ...msgs[0] });
   return msgs;
 }
+
+export async function createWrappedOnAptos(
+  client: AptosClient,
+  senderAddress: string,
+  tokenBridgeAddress: string,
+  tokenChain: ChainId,
+  tokenAddress: string,
```

Comment:
why to Address and Chain need to be specified? aren't they in the VAA?

Reply:
yes, that should be parsed off the vaa

Reply:
cc @heyitaki